db/migrate/20160112092557_add_valid_dates_to_business_entity.rb

```diff
@@ -0,0 +1,16 @@
+class AddValidDatesToBusinessEntity < ActiveRecord::Migration
+  def up
+    add_column :business_entities, :valid_from, :datetime
+    add_column :business_entities, :valid_to, :datetime
+    remove_index :business_entities, name: :unique_office_jurisdiction
+    add_index :business_entities, [:office_id, :jurisdiction_id, :valid_to], name: :unique_active_office_jurisdiction
```

Comment:
I added the minimum needed! [:office_id, :jurisdiction_id] was insufficient, adding valid_to ensured that an office/jurisdiction combination could be unique...